protostar-rust/src/lib.rs

```diff
@@ -1,36 +1,84 @@
-use pyo3::exceptions::RuntimeError;
-use pyo3::prelude::*;
-use pyo3::wrap_pyfunction;
-
-use anyhow::Context;
-use std::collections::HashMap;
-
-use cairo_lang_protostar::test_collector::collect_tests;
+use anyhow::{anyhow, Context, Result};
+use cairo_lang_protostar::test_collector::{collect_tests, LinkedLibrary};
 use cairo_lang_runner::{RunResultValue, SierraCasmRunner};
+use camino::{Utf8Path, Utf8PathBuf};
+use serde::Deserialize;
+use walkdir::WalkDir;
+
+#[derive(Deserialize, Debug)]
+pub struct ProtostarTestConfig {
+    #[serde(default)]
+    exit_first: bool,
+    #[serde(default)]
+    json: bool,
+    #[serde(default)]
+    last_failed: bool,
+    #[serde(default)]
+    report_slowest_tests: bool,
+}
 
 fn run_result_value_to_string(run_result: RunResultValue) -> String {
-    match run_result {
-        RunResultValue::Success(data) => {
-            return format!("Success {:?}", data);
-        }
-        RunResultValue::Panic(data) => {
-            return format!("Panic {:?}", data);
+    return match run_result {
+        RunResultValue::Success(data) => format!("Success {:?}", data),
+        RunResultValue::Panic(data) => format!("Panic {:?}", data),
+    };
+}
```

Comment:
```suggestion
fn run_result_value_to_string(run_result: RunResultValue) -> String {
    return match run_result {
        RunResultValue::Success(data) => format!("PASS {:?}", data),
        RunResultValue::Panic(data) => format!("FAIL {:?}", data),
    };
}
```
i know it will change but i am scared somebody will reuse those results